fdbserver/KeyValueStoreSQLite.actor.cpp

```diff
@@ -1431,7 +1431,14 @@ class KeyValueStoreSQLite : public IKeyValueStore {
 	KeyValueStoreSQLite(std::string const& filename, UID logID, KeyValueStoreType type, bool checkChecksums, bool checkIntegrity);
 	~KeyValueStoreSQLite();
 
-	Future<Void> doClean();
+	struct SpringCleaningWorkPerformed {
+		int lazyDeletePages;
+		int vacuumedPages;
+
+		SpringCleaningWorkPerformed() : lazyDeletePages(0), vacuumedPages(0) {}
```

Comment:
For future reference, you can just write

```
struct SpringCleaningWorkPerformed {
    int lazyDeletePages = 0;
    int vacuumedPages = 0;
};
```

And then the default constructor will initialize them accordingly.  This was introduced in C++11 as in-class initialization of non-static members (§12.6.2/8).

Reply:
Done.